cli/command/swarm/ca.go

```diff
@@ -61,6 +61,11 @@ func runRotateCA(dockerCli command.Cli, flags *pflag.FlagSet, opts caOptions) er
 	}
 
 	if !opts.rotate {
+		for _, f := range []string{flagCACert, flagCAKey, flagCACert, flagExternalCA} {
+			if flags.Changed(f) {
+				return fmt.Errorf("`--%s` flag ignored because you must `--rotate` to update the CA", f)
+			}
```

Comment:
I guess now that it's an error it's not ignored anymore. So maybe "--%s flag requires `--rotate` to update the CA" ?